es5-sham.js

```diff
@@ -190,6 +190,63 @@ if (!Object.create) {
                         // the following produces false positives
                         // in Opera Mini => not a reliable check
                         // Object.prototype.__proto__ === null
+
+    // Check for document.domain and active x support
+    // No need to use active x approach when document.domain is not set
+    // see https://github.com/es-shims/es5-shim/issues/150
+    // variation of https://github.com/kitcambridge/es5-shim/commit/4f738ac066346
+    /*global ActiveXObject */
+    var shouldUseActiveX = function shouldUseActiveX() {
+        var SUPPORTS_ACTIVEX;
+
+        try {
+            SUPPORTS_ACTIVEX = !!new ActiveXObject('htmlfile');
+        } catch (exception) {
+            SUPPORTS_ACTIVEX = false;
+        }
+
+        return SUPPORTS_ACTIVEX && document.domain;
```

Comment:
Should this `if (!document.domain) { return false; }` on the first line of the function, to avoid the try/catch when it's not set?